doc/application/application.rst

```diff
@@ -879,12 +894,8 @@ file.
 3. Otherwise, if a file :file:`prj.conf` exists in the application directory,
    the settings in it are used.
 
-4. Otherwise, there are no application-specific settings, and just the settings
-   in the :makevar:`BOARD`-specific configuration are used.
-
-Configuration settings that are specified in neither the :makevar:`BOARD`
-configuration file nor in the application configuration file(s) fall back on
-their default value, as given in the :file:`Kconfig` files.
```

Comment:
Why was the reference to the board configuration (`boards/ARCHITECTURE/BOARD/BOARD_defconfig`, etc.) removed here?

Think that would confuse me if I was new. Sounds like there's just `prj.config` and the default values from the Kconfig files.

Reply:
Because it was redundant.

> Sounds like there's just prj.conf and the default values from the Kconfig files.

No, read the docs from "The initial configuration for an application is produced by merging".

There it states that the board_defconfig is merged in.

Reply:
After reading through it I am not able to understand where this can be misread, can you describe where you would want the text?

Reply:
If you start reading at "the application-specific configuration file(s) can be specified in any of the
following ways", you end up at "configuration settings that have not been specified fall back on their default value, as given in the `Kconfig` files".

I think it might be easy to miss that this means "configuration settings that have not been specified in neither the `boards/ARCHITECTURE/BOARD/BOARD_defconfig` configuration file nor in any application-specific configuration file(s) fall back on their default value, as given in the Kconfig files".

Maybe the paragraph below could be changed to something like that, with point (4.) still removed, if you want to shorten it.